src/vmm/src/cpuid/intel/normalize.rs

```diff
@@ -296,211 +297,11 @@ impl super::IntelCpuid {
-    const DEFAULT_BRAND_STRING_BASE: &[u8] = b"Intel(R) Xeon(R) Processor @";
-
-    // The slice of the host string before the frequency suffix
-    // e.g. b"Intel(R) Xeon(R) Processor Platinum 8275CL CPU @ 3.00" and b"GHz"
-    let (before, after) = 'outer: {
-        for i in 0..host_brand_string.len() {
-            // Find position of b"THz" or b"GHz" or b"MHz"
-            if let [b'T' | b'G' | b'M', b'H', b'z', ..] = host_brand_string[i..] {
-                break 'outer Ok(host_brand_string.split_at(i));
-            }
-        }
-        Err(DefaultBrandStringError::MissingFrequency(host_brand_string))
-    }?;
-    debug_assert_eq!(
-        before.len().checked_add(after.len()),
-        Some(BRAND_STRING_LENGTH)
-    );
-
-    // We iterate from the end until hitting a space, getting the frequency number
-    // e.g. b"Intel(R) Xeon(R) Processor Platinum 8275CL CPU @ " and b"3.00"
-    let (_, frequency) = 'outer: {
-        for i in (0..before.len()).rev() {
-            let c = before[i];
-            match c {
-                b' ' => break 'outer Ok(before.split_at(i)),
-                b'0'..=b'9' | b'.' => continue,
-                _ => break,
-            }
-        }
-        Err(DefaultBrandStringError::MissingSpace(host_brand_string))
-    }?;
-    debug_assert!(frequency.len() <= before.len());
-
-    debug_assert!(
-        matches!(frequency.len().checked_add(after.len()), Some(x) if x <= BRAND_STRING_LENGTH)
-    );
-    debug_assert!(DEFAULT_BRAND_STRING_BASE.len() <= BRAND_STRING_LENGTH);
-    debug_assert!(BRAND_STRING_LENGTH.checked_mul(2).is_some());
-
-    // As `DEFAULT_BRAND_STRING_BASE.len() + frequency.len() + after.len()` is guaranteed
-    // to be less than or equal to  `2*BRAND_STRING_LENGTH` and we know
-    // `2*BRAND_STRING_LENGTH <= usize::MAX` since `BRAND_STRING_LENGTH==48`, this is always
-    // safe.
-    let len = DEFAULT_BRAND_STRING_BASE.len() + frequency.len() + after.len();
-
-    let brand_string = DEFAULT_BRAND_STRING_BASE
-        .iter()
-        .copied()
-        // Include frequency e.g. "3.00"
-        .chain(frequency.iter().copied())
-        // Include frequency suffix e.g. "GHz"
-        .chain(after.iter().copied())
-        // Pad with 0s to `BRAND_STRING_LENGTH`
-        .chain(
-            std::iter::repeat(b'\0').take(
-                BRAND_STRING_LENGTH
-                    .checked_sub(len)
-                    .ok_or(DefaultBrandStringError::Overflow)?,
-            ),
-        )
-        .collect::<Vec<_>>();
-    debug_assert_eq!(brand_string.len(), BRAND_STRING_LENGTH);
-
-    // SAFETY: Padding ensures `brand_string.len() == BRAND_STRING_LENGTH`.
-    Ok(unsafe { brand_string.try_into().unwrap_unchecked() })
-}
-
-#[cfg(test)]
-mod tests {
-    #![allow(
-        clippy::undocumented_unsafe_blocks,
-        clippy::unwrap_used,
-        clippy::as_conversions
-    )]
-
-    use super::*;
-
-    #[test]
-    fn default_brand_string_test() {
```

Comment:
are these gone?

Reply:
No, this was a mistake on my part in the rebase. Fixed.